qemu/tests/cfg/balloon_small_size.cfg

```diff
@@ -0,0 +1,12 @@
+- balloon_small_size:
+    virt_test_type = qemu
+    no Win2000
+    only Windows
```

Comment:
According to our discussion, your case combined two cases into one, "balloon_small_size" test and "balloon_boundary" test, and i suggested you to divide it into two cases.  This patch only cover the test case "ballloon_small_size", and the other patch covered "balloon_boundary" will send later. "balloon_small_size" only used for windows at present, as linux can't handle this issue at present. As " where the developer said they will realize it", it's not formal, but just maybe a future feature to make our product better. So, i suggest you'd better focus on present feature.
    If you think it's necessary to add subtest--reboot after this "small_size" test, i'll add it.
    Thanks for you reviewing and suggestion.